src/NuGet/Microsoft.Orleans.OrleansTelemetryConsumers.Counters.nuspec

```diff
@@ -1,33 +1,31 @@
-﻿<?xml version="1.0"?>
+<?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.Orleans.CounterControl</id>
```

Comment:
Please leave counter control in place and continue to support until installUtil is used in production environments.  Let's phase out counter control over time.  1.3 we support counter control or users can use installUtil.  1.3.*ish we depricate counter control.